BRAD/coder.py

```
@@ -201,19 +212,32 @@ def codeCaller(state):
+        try:
+            responseOriginal = {
+                'content' : response,
+                'time' : time.time() - start_time,
+                'call back': {
+                    "Total Tokens": cb.total_tokens,
+                    "Prompt Tokens": cb.prompt_tokens,
+                    "Completion Tokens": cb.completion_tokens,
+                    "Total Cost (USD)": cb.total_cost
+                }
+            }
+        except:
```

Ruff (0.7.0)

Do not use bare `except`

(E722)